mmv1/products/kms/CryptoKey.yaml

```
@@ -83,18 +83,13 @@ properties:
     name: 'labels'
     description: |
       Labels with user-defined metadata to apply to this resource.
-  - !ruby/object:Api::Type::Enum
+  - !ruby/object:Api::Type::String
     name: 'purpose'
     description: |
       The immutable purpose of this CryptoKey. See the
       [purpose reference](https://cloud.google.com/kms/docs/reference/rest/v1/projects.locations.keyRings.cryptoKeys#CryptoKeyPurpose)
       for possible inputs.
```

```suggestion
      The immutable purpose of this CryptoKey. See the
      [purpose reference](https://cloud.google.com/kms/docs/reference/rest/v1/projects.locations.keyRings.cryptoKeys#CryptoKeyPurpose)
      for possible inputs.
      Default value is "ENCRYPT_DECRYPT".
```
Only nit: Let's add the default value back to the field description for web doc. It seems it's auto generated for enum type but not strings